modules/EmailMan/EmailMan.php

```diff
@@ -1227,6 +1227,7 @@ private function shouldBlockEmail(SugarBean $bean)
         $result = $bean->db->query($query);
         $row = $bean->db->fetchByAssoc($result);
 
+        $emailOptInStatus = new \SuiteCRM\Enumerator\EmailOptInStatus();
```

Comment:
static class. Everything is public static or const. Whhy do you need an instance?

Reply:
We don't. I will use the class and reference through out this class